waspc/examples/todo-typescript/src/jobs/print.ts

```diff
@@ -1,8 +1,10 @@
 import { PrintTimeAndNumberOfTasks } from 'wasp/server/jobs'
+import { getTotalNumberOfTasks } from 'wasp/server/operations'
+
 export const printTimeAndNumberOfTasks: PrintTimeAndNumberOfTasks<
   {},
   void
 > = async (data, context) => {
-  const count = await context.entities.Task.count()
+  const count = await getTotalNumberOfTasks(null, context)
```

Comment:
We should probably rework this API when the function is void.

Reply:
What does this mean?

Reply:
It's kind of weird to ask users to send something as the first argument when the function doesn't actually need it.

Reply:
Ah okay, you meant when the args are void :D 

I agree 👍